cengage/react-magma
# Treeview > Docs > Selectable > Missing references

Issue: # [1531](https://github.com/cengage/react-magma/issues/1531)

## What I did
- Added references in Selectable example.

## Checklist 
- [N/A] I have added tests that prove my fix is effective or that my feature works

## How to test
- Go to Docs
- Open Treeview
- Scroll to Selectable example > Open in Codesandbox
- Interact with demo and verify no errors are thrown from missing references
